data_prep/reuters_graph_datasets.py

```diff
@@ -0,0 +1,208 @@
+from collections import defaultdict
```

Comment:
Is this a new file for safety reasons?

Reply:
yes indeed. In the end, we can delete the other file